dashboard/test/factories.rb

```diff
@@ -530,8 +530,7 @@
   factory :survey_result do
     user {create :teacher}
     kind 'Diversity2016'
-    properties {{survey2016_ethnicity_asian: "1"}}
-    properties {{survey2016_foodstamps: "3"}}
+    properties {{ethnicity_asian: "1", foodstamps: "3"}}
```

Comment:
Good catch, but it should be `survey_ethnicity_asian` and `survey_foodstamps` I believe.